pennylane/templates/embeddings/amplitude.py

```diff
@@ -215,17 +213,17 @@ def circuit(f=None):
     # normalize
     if isinstance(features[0], Variable):
         feature_values = [s.val for s in features]
-        norm = np.sum(np.abs(feature_values) ** 2)
+        norm = np.linalg.norm(feature_values, ord=2)
```

Comment:
Turned to use the same normalization method as used in `default.qubit` (happy for further suggestions, should be fine with the `math.isclose` checks if it's consistent at parts of the codebase).

Reply:
That's great!